charts/openvasd/README.md

```diff
@@ -0,0 +1,46 @@
+Contains the helm chart to deploy openvasd.
+
+# Install
+
+To install openvasd helm chart from a local path execute
+
+``` 
+helm install openvasd ./openvasd/ -f openvasd/values.yaml
+```
+
+You can also provide override the initial values within `openvasd/values.yaml` by providing an additional `-f` flag.
+
+As an example imagine you want to override the openvas image to your forked one you can create `~/openvasd.yaml` file containing: 
+
+```
+# Contains openvasd
+openvas:
+  repository: nichtsfrei/openvas-scanner
```

Comment:
```suggestion
  repository: greenbone/openvas-scanner
```
Not sure, if you should have your forked repository as example

Reply:
changed it to example